servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -128,29 +129,90 @@ def get_job(id, **kwargs):
+        start=_parse_datetime(latest_attempt.get('start')),
+        end=_parse_datetime(latest_attempt.get('end')),
+        stderr=latest_attempt.get('stderr'),
+        stdout=latest_attempt.get('stdout'),
+        inputs=update_key_names(latest_attempt.get('inputs', {})),
+        return_code=latest_attempt.get('returnCode'),
+        attempts=latest_attempt.get('attempt'),
+        call_root=latest_attempt.get('callRoot'),
+        job_id=latest_attempt.get('subWorkflowId'),
+        shard_statuses=None)
+
+
+def format_scattered_task(task_name, task_metadata):
+    temp_status_collection = {}
+    current_shard = ''
+    minStart = _parse_datetime(task_metadata[0].get('start'))
+    maxEnd = _parse_datetime(task_metadata[0].get('end'))
+
+    # go through calls in reverse to grab the latest attempt if there are multiple
+    # get earliest start time and latest end time
+    for shard in task_metadata[::-1]:
+        if current_shard != shard.get('shardIndex'):
+            status = task_statuses.cromwell_execution_to_api(
+                shard.get('executionStatus'))
+            if status not in temp_status_collection:
+                temp_status_collection[status] = 1
+            else:
+                temp_status_collection[
+                    status] = temp_status_collection[status] + 1
+        if minStart > _parse_datetime(shard.get('start')):
+            minStart = _parse_datetime(shard.get('start'))
+        if shard.get('executionStatus') not in ['Failed', 'Done']:
+            maxEnd = None
+        if maxEnd is not None and maxEnd < _parse_datetime(shard.get('end')):
+            maxEnd = _parse_datetime(shard.get('end'))
+        current_shard = shard.get('shardIndex')
+
+    shard_status_counts = [
+        ShardStatusCount(status=status, count=temp_status_collection[status])
+        for status in temp_status_collection
+    ]
+
+    # grab attempts, path and subWorkflowId from last call
+    return TaskMetadata(
+        name=remove_workflow_name(task_name),
+        execution_status=task_statuses.cromwell_execution_to_api(
+            task_metadata[-1].get('executionStatus')),
+        start=minStart,
+        end=maxEnd,
+        attempts=task_metadata[-1].get('attempt'),
+        call_root=remove_shard_path(task_metadata[-1].get('callRoot')),
+        job_id=task_metadata[-1].get('subWorkflowId'),
+        shard_statuses=shard_status_counts)
 
 
 def remove_workflow_name(name):
-    """ Remove the workflow name from the beginning of task, input and output names.
+    """ Remove the workflow name from the beginning of task, input and output names (if it's there).
     E.g. Task names {workflowName}.{taskName} => taskName
          Input names {workflowName}.{inputName} => inputName
          Output names {workflowName}.{taskName}.{outputName} => taskName.outputName
     """
-    return '.'.join(name.split('.')[1:])
+    if "." in name:
+        return '.'.join(name.split('.')[1:])
+    return name
```

Comment:
An optional nit that might save invisible computation time here by using [`str.partition()`](https://docs.python.org/3/library/stdtypes.html#str.partition): 
```
partitioned = name.partition('.')
name = partitioned[2] if partitioned[2] != '' else partitioned[0]
return name
```

Reply:
ok, done